mission_control/include/mission_control/mission_control.h

```diff
@@ -103,24 +106,19 @@ class MissionControlNode
 
   ros::Timer reportExecuteMissionStateTimer;
   Mission::MissionState last_state;
-  int last_id;
 
   ros::Timer reportHeartbeatTimer;
+  ros::Timer executeMissionTimer;
 
  private:
   // Vars holding runtime params
-  std::string mission_path;
-  bool disable_abort;
   double reportExecuteMissionStateRate;
   double reportHeartbeatRate;
 
-  pose_estimator::CorrectedData m_correctedData;
-
   int m_current_mission_id;
   int m_mission_id_counter;
-  Mission* m_cur_mission;
 
-  std::map<int, Mission*> m_mission_map;
+  std::unordered_map<int, std::shared_ptr<Mission>> m_mission_map;
```

Comment:
@patapufate nit: why not an `std::unique_ptr`?

Reply:
fixed